nss/integration-tests/integration_test.go

```diff
@@ -20,6 +24,24 @@ func TestIntegration(t *testing.T) {
 
 	buildRustNSSLib(t)
 
+	// Create a default daemon to use for most test cases.
+	defaultSocket := "/tmp/nss-integration-tests.sock"
+	defaultDbState := "multiple_users_and_groups"
+	defaultOutputPath := filepath.Join(filepath.Dir(daemonPath), "gpasswd.output")
+	defaultGroupsFilePath := filepath.Join(testutils.TestFamilyPath(t), "gpasswd.group")
+
+	ctx, cancel := context.WithCancel(context.Background())
+	_, stopped := testutils.RunDaemon(ctx, t, daemonPath,
+		testutils.WithSocketPath(defaultSocket),
+		testutils.WithPreviousDBState(defaultDbState),
+		testutils.WithEnvironment(gpasswdMockEnv(t, defaultOutputPath, defaultGroupsFilePath)...),
+	)
+
+	t.Cleanup(func() {
+		cancel()
+		<-stopped
+	})
```

Comment:
@didrocks, here is the default daemon running.

Reply:
Thanks for pointing this out!